src/pages/gateway/create-mesh.md

```diff
@@ -80,6 +80,27 @@ When creating or updating a mesh, the file to upload must have the `.json` filen
 
 Each workspace within a project can only have one mesh associated with it at a time.
 
+## Mesh example
```

Comment:
Can we move this example above the command and move the existing one down below?